src/java/org/apache/cassandra/db/compaction/AbstractCompactionTask.java

```diff
@@ -138,8 +145,9 @@ public int execute()
 
     private Throwable cleanup(Throwable err)
     {
+        Throwable taskErr = err;
```

Comment:
`taskErr` is only used for comparison to my understanding. I suggest to have a Boolean variable here, e.g., `isSuccess`, and store the value. It will make code more readable. `final` can also help.

Reply:
I was not exactly clear about these specific changes either, but made the decision to port them as they were originally made because there are later cndb ticket changes to this class and specific code and if we port those later it could be confusing if the previous state is already changed.
I know that sometimes we include the most recent code if there have been changes since the original, but in this case I noted four additional cndb tickets which include changes to this class and again it seemed to me more clear if each change would be made in order rather than taking one piece from the latest code.

Reply:
Oh, sorry, I made this comment in the wrong location. This was intended to be about changes in the `execute` method, while your comment is about the `cleanup` - apologies for the mixup.

You make a good suggestion and I will apply it.